mobile/native/src/commons/api.rs

```diff
@@ -26,3 +27,18 @@ impl From<Price> for trade::Price {
         }
     }
 }
+
+pub struct ChannelInfo {
+    /// The total capacity of the channel as defined by the funding output
+    pub channel_capacity: u64,
+    pub reserve: Option<u64>,
+}
```

Comment:
🤔 could the remaining free functions be part of channel info?

-`coordinator_liquidity_multiplier`
- `max_channel_value`
- `contract_tx_fee_rate?`

I guess then the capacity would need to become an `Option` (and lack of capacity would need that there's no channel that's available & ready).

Reply:
Would address this in a follow up - the current solution works and can be improved.